.changeset/quick-ads-pretend.md

```diff
@@ -0,0 +1,6 @@
+---
+"@namehash/namekit-react": minor
+"@namehash/ens-utils": minor
+---
+
+Create DisplayedPrice and DisplayedPriceWithSymbol components + new price utilities
```

Comment:
```suggestion
Create DisplayedPrice and DisplayedPriceConversion components + new price utilities
```

Also, shouldn't there be something in this changeset description about `CurrencySymbol` too?